packages/amplify-codegen-e2e-tests/src/__tests__/feature-flags.test.ts

```diff
@@ -5,6 +5,7 @@ import { pathManager } from 'amplify-cli-core';
 const codegenFeatureFlags = {
     "handlelistnullabilitytransparently": true,
     "addtimestampfields": true,
+    "generateIndexRules": true,
```

Comment:
PR to add new feature flag: https://github.com/aws-amplify/amplify-cli/pull/7446